mtags/src/main/scala-3/scala/meta/internal/pc/completions/Completions.scala

```diff
@@ -176,24 +176,26 @@ class Completions(
     val (all, result) =
       if exclusive then (advanced, SymbolSearch.Result.COMPLETE)
       else
+        val keywords = KeywordsCompletions.contribute(path, completionPos)
```

Comment:
Moved it here, so if advanced completions are not `exclusive`, keyword completions are also shown. It is needed in case like 
```scala 
trait A {
  def value(): Unit
}
object B. extends A {
  va@@
}
```
We would want to see all `val`, `var` and `override def value`